wafer_space/shuttles/migrations/0002_alter_shuttle_name.py

```
@@ -0,0 +1,24 @@
+# Generated by Django 5.2.6 on 2025-12-02 10:51
+
+import wafer_space.shuttles.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("shuttles", "0001_initial"),
+    ]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)